src/core/layout/navbar/responsive-drawer/icon.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { useCallback } from 'react'
+import { IconButton } from '@material-ui/core'
+import MenuIcon from '@material-ui/icons/Menu'
+
+import { useContext } from 'core/layout/navbar/context'
+import { useStyles } from 'core/layout/navbar/responsive-drawer/style'
+
+const useOnClick = () => {
```

Comment:
Let's extract this hook to a separate file.

Reply:
Done